pkg/vcs/vcs.go

```diff
@@ -25,8 +25,20 @@ func VCS(path string) (result v1.VCS) {
 		return
 	}
 
-	return v1.VCS{
+	result = v1.VCS{
 		Revision: head.Hash().String(),
 		Modified: !s.IsClean(),
 	}
+
+	// Set optional remotes field
+	remotes, err := repo.Remotes()
```

Comment:
For my own education, what would be the usecase to know where is the remote in the git repo that image is built?

Reply:
So, including remotes means that, in some cases, the HEAD can be traced back to an actual git object. I think there are quite a few usecases that can make use of this, but the immediate example for me is `AppImagePullSuccess`; in which we track the VCS info for images both before and after a successful pull. Given both, a user could potentially produce a source diff to make the update easier to grok.